docs/components/modal.md

```diff
@@ -352,6 +352,20 @@ Maximum height of the content area. When content overflows, content area becomes
 </KModal>
 ```
 
+### zIndex
+
+`z-index` value for the modal. Defaults to `1100`.
+
+```html
+<KModal
+  z-index="9999"
+  title="Modal"
+  :visible="modalVisible"
+  @cancel="handleModalClose"
+  @proceed="handleModalProceed"
+/>
+```
+
```

Comment:
Since a custom z-index won't be visually discernable, there's not much benefit of adding a try-out component.